File: Sakila_Net/Sakila_Net/CS_SakilaLib/DAO/DAOFilmActor.cs
//using System;
using System.Data;
//using System.Collections.Generic;
//using System.Configuration;
//using System.Collections;
using Cl.Harrys.Sakila.Model;
namespace  Cl.Harrys.Sakila.DAO
{   
  public class DAOFilmActor 
  {
       public static DataTable sqlSelectAll(int nPagina,int cantidad)
	   {
           return DAOBD.DAOFilmActor.selectAll(nPagina,cantidad);
	   }
       public static bool sqlSelect(ClFilmActor filmActor)
	   {
            return DAOBD.DAOFilmActor.SelectOne(filmActor); 
	   }
	   public static int sqlInsert(ClFilmActor filmActor)
	   {
            return DAOBD.DAOFilmActor.sqlInsert(filmActor);
       }
	   public static int sqlDelete(ClFilmActor filmActor)
	   {
            return DAOBD.DAOFilmActor.sqlDelete(filmActor);
       }
	   public static int sqlUpdate(ClFilmActor filmActor)
	   {
            return DAOBD.DAOFilmActor.sqlUpdate(filmActor);
       }
   }
}

File: Sakila_Net/Sakila_Net/VB_SakilaLib/DAO/DAOFilmActor.vb
'using System;
Imports  System.Data
'using System.Collections.Generic;
'using System.Configuration;
'using System.Collections;
Imports  SakilaLib.Cl.Harrys.Sakila.Model
'using Cl.Harrys.BD;

Namespace Cl
    Namespace Harrys
        Namespace Sakila
            Namespace DAO

                Public Class DAOFilmActor
                    'Inherits SakilaLib.Cl.Harrys.Sakila.DAOBD.DAOFilmActor
                    Public Shared  Function SelectAll(nPagina as Integer ,cantidad as Integer) as DataTable
                        Return DAOBD.DAOFilmActor.SelectAll(nPagina, cantidad)	   
                    End Function
                    Public Shared  Function SelectOne( filmActor as ClFilmActor) as Boolean
                        Return DAOBD.DAOFilmActor.SelectOne(filmActor)     
                    End Function
                    Public Shared  Function SqlInsert( filmActor as ClFilmActor) as Integer
                        Return DAOBD.DAOFilmActor.SqlInsert(filmActor)     
                    End Function
                    Public Shared  Function SqlDelete( filmActor as ClFilmActor) as Integer
                        Return DAOBD.DAOFilmActor.SqlDelete(filmActor)     
                    End Function
                    Public Shared  Function SqlUpdate( filmActor as ClFilmActor) as Integer
                        Return DAOBD.DAOFilmActor.SqlUpdate(filmActor)     
                    End Function
                End Class

            End Namespace  
        End Namespace  
    End Namespace  
End Namespace  
